### PR TITLE
New instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Includes cal csvs for the new PRTSZA instruments; all coefficients are constant (initial template & coefficients provided by Andrew).  The date used in filenames is the date we received the instruments (in WHOI_Asset_Tracking).  No source_file reference - these are one-offs plus the vendor didn't provide any documentation.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

**Comment by rtravis-whoi** [comment]
[New Instruments - Cal CSVS needed [DT-251]](https://trello.com/c/B3PlliRC/251-new-instruments-cal-csvs-needed-dt-251)

File: calibration/SPKIRB/CGINS-SPKIRB-00273__20240718.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+273,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]",source_file SPKIR-B_OCR-507_SN_273_Calibration_Files_2024-07-18.zip > DI7273F.cal
+273,CC_offset,"[2147431112.2, 2147304478.4, 2146979315.1, 2147310340.6, 2147938306.9, 2147823672.1, 2146926236.6]",
+273,CC_scale,"[2.02145430622e-007, 1.98628126725e-007, 2.07835859093e-007, 2.00043892225e-007, 2.14090404297e-007, 2.06438176597e-007, 1.96928436534e-007]",
```
Comment (cooleyky) [inline calibration/SPKIRB/CGINS-SPKIRB-00273__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00288__20240725.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+288,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]",source_file SPKIR-B_OCR-507_SN_288_Calibration_Files_2024-07-25.zip > DI7288G.cal
+288,CC_offset,"[2148996236.9, 2147207159.8, 2147380076.7, 2147182572.7, 2147387266.5, 2147697369.0, 2147102843.4]",
+288,CC_scale,"[1.88915549989e-007, 1.84638024812e-007, 2.17100109583e-007, 1.90420888725e-007, 1.92356021992e-007, 2.04992136718e-007, 1.9513268078e-007]",
```
Comment (cooleyky) [inline calibration/SPKIRB/CGINS-SPKIRB-00288__20240725.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00289__20240718.csv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+289,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]",source_file SPKIR-B_OCR-507_SN_289_Calibration_Files_2024-07-18.zip > DI7289G.cal
+289,CC_offset,"[2147006776.1, 2147191077.5, 2148077324.5, 2147672116.9, 2147294179.8, 2147606466.5, 2147372586.9]",
+289,CC_scale,"[2.17875653095e-007, 2.08109379829e-007, 2.04026572552e-007, 2.11993186866e-007, 1.9562625735e-007, 2.13461976893e-007, 1.97311127635e-007]",
```
Comment (cooleyky) [inline calibration/SPKIRB/CGINS-SPKIRB-00289__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00290__20240718.csv [hunk 4]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+290,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR-B_OCR-507_SN_290_Calibration_Files_2024-07-18.zip > DI7290E.cal
+290,CC_offset,"[2147767484.4, 2147941666.3, 2147379004.1, 2147004253.0, 2147945493.1, 2147560150.5, 2147150228.4]",
+290,CC_scale,"[2.22257039486e-007, 1.96598327613e-007, 2.04103973585e-007, 2.07798036554e-007, 1.91635803326e-007, 1.95049666139e-007, 1.86232256957e-007]",
```
Comment (cooleyky) [inline calibration/SPKIRB/CGINS-SPKIRB-00290__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00291__20240718.csv [hunk 5]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+291,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR-B_OCR-507_SN_291_Calibration_Files_2024-07-18.zip > DI7291E.cal
+291,CC_offset,"[2147925762.9, 2147666421.6, 2148106602.9, 2147145583.2, 2147194211.8, 2146744597.8, 2147669689.6]",
+291,CC_scale,"[2.18458908911e-007, 2.04010241267e-007, 2.16010112835e-007, 2.07471079329e-007, 1.99271962124e-007, 2.02400796015e-007, 2.14407989373e-007]",
```
Comment (cooleyky) [inline calibration/SPKIRB/CGINS-SPKIRB-00291__20240718.csv:4]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02253__20230719.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2253,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2253,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02253__20230719.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02257__20230918.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2257,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2257,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02257__20230918.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02258__20230918.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2258,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2258,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02258__20230918.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02261__20230918.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2261,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2261,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02261__20230918.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02262__20230918.csv [hunk 10]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2262,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2262,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02262__20230918.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02263__20230918.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2263,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2263,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02263__20230918.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02265__20230918.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2265,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2265,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02265__20230918.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02266__20231024.csv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2266,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2266,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02266__20231024.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02267__20231024.csv [hunk 14]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2267,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2267,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02267__20231024.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02268__20231024.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2268,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2268,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02268__20231024.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02269__20231024.csv [hunk 16]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2269,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2269,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02269__20231024.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02270__20231024.csv [hunk 17]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2270,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2270,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02270__20231024.csv:3]: looks good

File: calibration/PRTSZA/CGINS-PRTSZA-02271__20231024.csv [hunk 18]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+2271,CC_lower_bin_size,"[1.00,1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00]",constant
+2271,CC_upper_bin_size,"[1.48,1.74,2.05,2.42,2.86,3.38,3.98,4.70,5.55,6.55,7.72,9.12,10.80,12.70,15.00,17.70,20.90,24.60,29.10,34.30,40.50,47.70,56.30,66.50,78.40,92.60,109.00,129.00,152.00,180.00,212.00,250.00,297.00,354.00,420.00,500.00]",constant
```
Comment (sbeaulieu) [inline calibration/PRTSZA/CGINS-PRTSZA-02271__20231024.csv:3]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00230__20240718.csv [hunk 19]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+230,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]",source_file SPKIR-B_OCR-507_SN_230_Calibration_Files_2024-07-18.zip > DI7230E.cal
+230,CC_offset,"[2147292839.7, 2147447742.9, 2148021885.5, 2147717145.0, 2147606279.5, 2147433909.3, 2147799858.1]",
+230,CC_scale,"[2.04862878791e-007, 1.96350339605e-007, 2.21057130779e-007, 2.0175415512e-007, 2.06631350795e-007, 1.95753565238e-007, 2.29406506902e-007]",
```
Comment (sbeaulieu) [inline calibration/SPKIRB/CGINS-SPKIRB-00230__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00273__20240718.csv (hunk 1 above)
Comment (sbeaulieu) [inline calibration/SPKIRB/CGINS-SPKIRB-00273__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00288__20240725.csv (hunk 2 above)
Comment (sbeaulieu) [inline calibration/SPKIRB/CGINS-SPKIRB-00288__20240725.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00289__20240718.csv (hunk 3 above)
Comment (sbeaulieu) [inline calibration/SPKIRB/CGINS-SPKIRB-00289__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00290__20240718.csv (hunk 4 above)
Comment (sbeaulieu) [inline calibration/SPKIRB/CGINS-SPKIRB-00290__20240718.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00291__20240718.csv (hunk 5 above)
Comment (sbeaulieu) [inline calibration/SPKIRB/CGINS-SPKIRB-00291__20240718.csv:4]: looks good

File: calibration/ADCPTQ/CGINS-ADCPTQ-01035__20240610.csv [hunk 20]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1035,CC_scale_factor1,0.45,constant; source_file ADCPT-Q_WHS600-200_SN_1035_Inspection_and_Test_Report_2024-06-10.pdf
+1035,CC_scale_factor2,0.45,constant
+1035,CC_scale_factor3,0.45,constant
+1035,CC_scale_factor4,0.45,constant
```
Comment (sbeaulieu) [inline calibration/ADCPTQ/CGINS-ADCPTQ-01035__20240610.csv:5]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20240725.csv [hunk 21]
```diff
@@ -0,0 +1,8 @@
...
..., 271.85, 272.66, 273.47, 274.27, 275.08, 275.89, 276.7, 277.51, 278.32, 279.13, 279.94, 280.75, 281.56, 282.37, 283.18, 283.99, 284.8, 285.61, 286.42, 287.23, 288.03, 288.84, 289.65, 290.46, 291.27, 292.08, 292.89, 293.7, 294.51, 295.32, 296.12, 296.93, 297.74, 298.55, 299.36, 300.17, 300.98, 301.78, 302.59, 303.4, 304.21, 305.02, 305.82, 306.63, 307.44, 308.25, 309.06, 309.86, 310.67, 311.48, 312.29, 313.09, 313.9, 314.71, 315.51, 316.32, 317.13, 317.93, 318.74, 319.55, 320.35, 321.16, 321.97, 322.77, 323.58, 324.38, 325.19, 326, 326.8, 327.61, 328.41, 329.22, 330.02, 330.83, 331.63, 332.44, 333.24, 334.04, 334.85, 335.65, 336.46, 337.26, 338.06, 338.87, 339.67, 340.47, 341.28, 342.08, 342.88, 343.68, 344.49, 345.29, 346.09, 346.89, 347.69, 348.5, 349.3, 350.1, 350.9, 351.7, 352.5, 353.3, 354.1, 354.9, 355.7, 356.5, 357.3, 358.1, 358.9, 359.7, 360.5, 361.29, 362.09, 362.89, 363.69, 364.49, 365.28, 366.08, 366.88, 367.67, 368.47, 369.27, 370.06, 370.86, 371.66, 372.45, 373.25, 374.04, 374.84, 375.63, 376.42, 377.22, 378.01, 378.81, 379.6, 380.39, 381.19, 381.98, 382.77, 383.56, 384.35, 385.15, 385.94, 386.73, 387.52, 388.31, 389.1, 389.89, 390.68, 391.47, 392.26, 393.05, 393.84]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01094__20240725.csv:8]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20240724.csv [hunk 22]
```diff
@@ -0,0 +1,8 @@
...
...272.64, 273.44, 274.25, 275.05, 275.85, 276.66, 277.46, 278.26, 279.07, 279.87, 280.67, 281.48, 282.28, 283.08, 283.89, 284.69, 285.49, 286.3, 287.1, 287.91, 288.71, 289.51, 290.32, 291.12, 291.92, 292.73, 293.53, 294.33, 295.14, 295.94, 296.74, 297.55, 298.35, 299.15, 299.96, 300.76, 301.56, 302.37, 303.17, 303.97, 304.78, 305.58, 306.38, 307.19, 307.99, 308.79, 309.59, 310.4, 311.2, 312, 312.81, 313.61, 314.41, 315.21, 316.02, 316.82, 317.62, 318.42, 319.22, 320.03, 320.83, 321.63, 322.43, 323.23, 324.04, 324.84, 325.64, 326.44, 327.24, 328.04, 328.84, 329.64, 330.45, 331.25, 332.05, 332.85, 333.65, 334.45, 335.25, 336.05, 336.85, 337.65, 338.45, 339.25, 340.05, 340.85, 341.65, 342.45, 343.25, 344.04, 344.84, 345.64, 346.44, 347.24, 348.04, 348.84, 349.63, 350.43, 351.23, 352.03, 352.82, 353.62, 354.42, 355.22, 356.01, 356.81, 357.61, 358.4, 359.2, 360, 360.79, 361.59, 362.38, 363.18, 363.97, 364.77, 365.56, 366.36, 367.15, 367.95, 368.74, 369.53, 370.33, 371.12, 371.92, 372.71, 373.5, 374.29, 375.09, 375.88, 376.67, 377.46, 378.26, 379.05, 379.84, 380.63, 381.42, 382.21, 383, 383.79, 384.58, 385.37, 386.16, 386.95, 387.74, 388.53, 389.32, 390.11, 390.9, 391.68, 392.47, 393.26]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01103__20240724.csv:8]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20240725.csv [hunk 23]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20240725.csv:8]: looks good

**Review by sbeaulieu — APPROVED** [review]

**Pull request merged**